start-server-and-test.js

```diff
@@ -0,0 +1,102 @@
+const { spawn } = require('child_process');
+const fetch = require('node-fetch');
+
+/* eslint-disable no-console */
+
+const serverUrl = 'http://localhost:4000';
```

Comment:
For these types of constants, we use to name them in caps like `SERVER_URL` or `SERVER_TIMEOUT`

Reply:
Sure, will keep that in mind.